onnx/checker.h

```diff
@@ -27,12 +28,27 @@ class ValidationError : public std::runtime_error {
 #define fail_check(...) \
   throw onnx::checker::ValidationError(onnx::MakeString(__VA_ARGS__));
 
+struct CheckerContext {
+  int ir_version;
+  std::unordered_map<std::string, int> opset_imports;
+  int get_ir_version() { return ir_version; }
+  void set_ir_version(int v) { ir_version = v; }
+  // NB: Does a copy
```

Comment:
As you commented here, I'm wondering why we not return const reference please :).

Reply:
It looks like pybind11 can handle it correctly, so reference it is.